README.md

```diff
@@ -1,2 +1,30 @@
-# postgres
-Work in progress, not stable, expect force pushes of this repo
+# Autopilot Pattern Postgres
+
+Postgres designed for automated operation using the [Autopilot Pattern](http://autopilotpattern.io/). This repo serves as a blueprint demonstrating the pattern.
```

Comment:
I think this could be restructured:

> This repo serves as a blueprint demonstrating [Postgres](https://www.postgresql.org/) designed for automated operation using the [Autopilot Pattern](http://autopilotpattern.io/).